Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:2.7-alpine
 WORKDIR /src
 RUN pip install pipenv
 COPY Pipfile Pipfile.lock /src/
```

Comment:
We should probably remove the Pipfile.lock if we're not using it here.

Reply:
Fixed.